controllers/informationCenter/SubmissionInformationCenterHandler.inc.php

```diff
@@ -20,6 +20,22 @@
 class SubmissionInformationCenterHandler extends InformationCenterHandler {
 
 	/**
+	 * @copydoc InformationCenterHandler::viewInformationCenter()
+	 */
+	function viewInformationCenter($args, $request) {
+		$this->setupTemplate($request);
```

Comment:
the parent class will run this when it's called on line 35, so it can be removed here.